api/hasura/event-triggers/check-nominee-discord.ts

```diff
@@ -0,0 +1,20 @@
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+
+import { EventTriggerPayload } from '../types';
+
+import handleCheckNomineeMsg from './utils/handleCheckNomineeMsg';
+
+export default async function handler(req: VercelRequest, res: VercelResponse) {
```

Comment:
I noticed that topocount made a verification middleware, I think we will want the same one here before merging

Reply:
we can let @topocount 's PR merge first to main, so @jsur can easily just pull in the changes

Reply:
🆒